superset/db_engine_specs/base.py

```diff
@@ -137,6 +137,7 @@ class BaseEngineSpec:  # pylint: disable=too-many-public-methods
     """Abstract class for database engine specific configurations"""
 
     engine = "base"  # str as defined in sqlalchemy.engine.engine
+    engine_name = "Base Database"
```

Comment:
Can this be an optional `str`? No one should be instantiating the base class.

Reply:
sgtm, simply removing this line should be fine then right?

```suggestion
```